frontend/src/static/js/v_summary.js

```diff
@@ -117,6 +117,7 @@ window.vSummary = {
       minDate: '',
       maxDate: '',
       contributionBarColors: {},
+      errorMessages: window.app.errorMessages,
```

Comment:
Perhaps it might be better to pass `errorMessages` as a prop instead from `window.app` vue object to `vSummary`

Reply:
This solution somehow dosen't work

Reply:
@jamessspanggg 

Can take a look?

Reply:
Yes, you can pass the `errorMessages` as a prop using `v-bind`.